crates/dapf/src/cli_parsers.rs

```diff
@@ -0,0 +1,324 @@
+//! Human friendly parsers for common types of parameters to DAP functions.
+// Copyright (c) 2024 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+use std::{
+    fmt,
+    io::{self, IsTerminal as _},
+    ops::ControlFlow,
+    str::FromStr,
+};
+
+use anyhow::{anyhow, Context};
+use clap::{builder::PossibleValue, ValueEnum};
+use daphne::{
+    hpke::HpkeKemId,
+    messages::{Base64Encode, TaskId},
+    vdaf::{Prio3Config, VdafConfig},
+    DapQueryConfig,
+};
+
+#[derive(Debug, Clone, Copy, Default)]
+pub enum DefaultVdafConfigs {
+    Prio2Dimension99k,
+    #[default]
+    Prio3NumProofs2,
+    Prio3NumProofs3,
```

Comment:
These names are very strange and need documentation. For each variant, I would spell out what VDAF is in use and what are its parameters.

I also think it's worth saying something like "these variants are used for load testing".

Reply:
BTW I see this code is just moved, but we might as well add some documentation!

Reply:
I just wanted some quick default vdafs to pass through the commandline. So I wanted the names to be short which means they won't be very good either

Reply:
The names are find, just document :)